### ISSUE TITLE
Typo in HTTP Cookies, "Updating cookies via JavaScript" Section

### ISSUE THREAD

**Issue by At11011** [mdn/content#34121]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies

### What specific section or headline is this issue about?

Updating cookies via JavaScript

### What information was incorrect, unhelpful, or incomplete?

There is a small typo in this section:

```
You can also access existing cookies and set new values for them, provided the [HttpOnly](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#httponly) attribute isn't set on them (i.e. in the Set-Cookie header than created it):
```

This is the typo:

```
(i.e. in the Set-Cookie header ___than___ created it)
```

### What did you expect to see?

In this text:

"(i.e. in the Set-Cookie header than created it)"

"than" should be replaced with "that".

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/http/cookies`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/http/cookies/index.md
* Last commit: https://github.com/mdn/content/commit/b557aa07561984cf5eb008d4c4e53804222e387f
* Document last modified: 2024-05-25T03:43:02.000Z

</details>

**Comment by bsmth** [comment]
Thanks for raising this one. Would you like to open a PR to fix it?

**Comment by At11011** [comment]
> Thanks for raising this one. Would you like to open a PR to fix it?

Sure thing, I opened one here:
https://github.com/mdn/content/pull/34131

Thanks!

**Comment by Josh-Cena** [comment]
@At11011 FYI, the usual workflow is this: you open an issue -> you send a PR -> in the PR description, you say "fixes https://github.com/mdn/content/issues/34121" -> when that PR gets merged, this issue will be automatically closed. You only close the issue when the issue is actually fixed, not when the PR is still open.

**Comment by At11011** [comment]
> @At11011 FYI, the usual workflow is this: you open an issue -> you send a PR -> in the PR description, you say "fixes #34121" -> when that PR gets merged, this issue will be automatically closed. You only close the issue when the issue is actually fixed, not when the PR is still open.

Thank you, I'll do that next time.

That was actually my first pull request to a large public repo, I appreciate your guidence and promptness.
